src/analysis/functions/infer_regusage.rs

```diff
@@ -4,10 +4,11 @@ use analysis::inst_combine;
 use frontend::radeco_containers::{RadecoFunction, RadecoModule};
 use middle::dce;
 use middle::ir;
-use middle::regfile::{RegisterId, RegisterUsage, SubRegisterFile};
+use middle::regfile::*;
```

Comment:
I think you should explicitly spell out what you want to import. This helps:
1. Makes it clear what's being imported
2. In case more things are added to `regfile`, they won't "automatically" be imported here.